.circleci/config.yml

```diff
@@ -122,22 +122,65 @@ commands:
           name: Install libvips
           command: sudo apt-get install -y libvips
 
-jobs:
-  persist_version:
-    executor: base
+  install_solidus:
+    parameters:
+      flags:
+        type: string
+        default: "# no options"
+        description: "flags to be passed to `bin/rails g solidus:install"
+      run_specs:
+        type: boolean
+        default: false
     steps:
-      - setup
-
       - run:
-          name: Persist Solidus version
+          name: "Cleanup & check rails version"
           command: |
-            mkdir -p workspace
-            bundle exec ruby -e "puts Spree.solidus_gem_version.segments[0..1].join('.')" > workspace/solidus-version
-
-      - persist_to_workspace:
-          root: workspace
+            rm -rf /tmp/my_app # cleanup previous runs
+            gem search -eq rails > /tmp/rails-version # get the latest rails from rubygems
+            gem search -eq solidus > /tmp/solidus-version # get the latest solidus from rubygems
+            cat /tmp/rails-version
+            cat /tmp/solidus-version
+      - restore_cache:
+          keys:
+            - solidus-installer-v3-{{ checksum "/tmp/rails-version" }}-{{ checksum "/tmp/solidus-version" }}
+      - run:
+          name: "Install rails"
+          command: |
+            bundle config set path '/tmp/bundle'
+            # Preinstall the latest rails and solidus in order to have most of the dependencies cached
+            gem install rails solidus --conservative
+      - run:
+          name: "Prepare the rails application"
+          command: |
+            cd /tmp
+            test -d my_app || rails new my_app --skip-javascript --skip-git
+      - save_cache:
+          key: solidus-installer-v2-{{ checksum "/tmp/rails-version" }}
           paths:
-            - solidus-version
+            - /tmp/my_app
+            - /tmp/bundle
+      - run:
+          name: "Run `solidus:install` with `<<parameters.flags>>`"
+          command: |
+            cd /tmp/my_app
+            bundle add solidus --git "file://$(ruby -e"puts File.expand_path ENV['CIRCLE_WORKING_DIRECTORY']")"
+            export SKIP_SOLIDUS_BOLT='true' # workaround for solidus_frontend not respecting --auto-accept when asking about bolt
```

Comment:
Now that we're respecting `--auto-accept`, we can:

- Remove the comment and keep skipping solidus_bolt.
- Remove the line and install solidus_bolt.